documentation/assemblies/configuring/assembly-config.adoc

```diff
@@ -3,7 +3,7 @@
 // configuring/configuring.adoc
 
 [id='overview-{context}']
-= Configuring a deployment
+= Configuring a Strimzi deployment
```

Comment:
This seems a bit weird name. What does it mean? `Strimzi Cluster Operator deployment`? Or the _operands_?

Reply:
Reverted